direct/common/subsample.py

```diff
@@ -87,7 +85,7 @@ def choose_acceleration(self):
         raise NotImplementedError("Uniform range is not yet implemented.")
 
     @abstractmethod
-    def mask_func(self, shape):
+    def mask_func(self, shape, return_acs=False, seed=None):
         raise NotImplementedError("This method should be implemented by a child class.")
```

Comment:
I don't think this is needed when you use a `@abstractmethod` decorator. You can just add a comment instead describing what it is.

Reply:
Codacy was complaining about this.